Makefile

```
@@ -1,4 +1,4 @@
-.PHONY: proto proto-check install test test-unit test-integration lint clean
+.PHONY: proto proto-check install install-uv test test-unit test-integration lint clean
```

checkmake (0.2.2)

[warning] Missing required phony target "all"

(minphony)